samples/aml_monitoring/local.settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "",
```

Comment:
What kind of error message will a user get if they forget to fill this in? I'm wondering whether a blank value or a placeholder value might create a better experience.

Reply:
In both cases, they seem to get the following exception:

```
Missing value for AzureWebJobsStorage in local.settings.json. This is required for all triggers other than httptrigger, kafkatrigger. You can run 'func azure functionapp fetch-app-settings <functionAppName>' or specify a connection string in local.settings.json.
```

But I think the empty string is the more future-proof approach, so I'm opting for it :)